### DISCUSSION TITLE
Does master branch supports Type of Any

### DISCUSSION THREAD

**Discussion by zylantian** [protobuf-c/protobuf-c#626] [Q&A] [answered]
Type of Any is protobuf3's new feature, I used master branch and next branch, but maybe all not support this. Is that right?

**Comment by Anda001** [comment]
can the author  give a response?

**Comment by lipnitsk** [comment, marked as answer]
protobuf-c does not do anything special with `Any` types. They are supported, since [`Any`][0] is just a `string` URL and a `bytes` field, but it is up to the caller to resolve the URL and to pass the correct `ProtobufCMessageDescriptor` into the unpack function to unpack the `bytes` field.

[0]: https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/any.proto
